.github/ISSUE_TEMPLATE/proposal_technical.yml

```
@@ -0,0 +1,255 @@
+      multiple: true
+      options:
+        - no other tool
+        - one more tool 
+        - a few more tools
+        - an entire tool group
+        - might propagate to all tools
+        
+  - type: textarea
+    attributes:
+      label: Other tools that get touched
+      description: If you put the Impact Scale to anything but None, list all the tools / repos that get affected.
+
+  - type: markdown
+    attributes:
+      value: |
+        # Implementation and Details
+
+  - type: textarea
+    attributes:
+      label: How it Solves the Problem
+      description: Explain how the proposed implementation addresses the initial problems.
+
+  - type: textarea
+    attributes:
+      label: Implementation Idea/Details
+      description: Provide a detailed plan for implementing the project or feature.
+
+  - type: textarea
+    attributes:
+      label: Technical Approach
+      description: Describe the technical steps, methodologies, and tools used.
+
+  - type: textarea
+    attributes:
+      label: Architecture
+      description: Outline the high-level architecture and design of the proposed solution.
+
+  - type: textarea
+    attributes:
+      label: Unresolved Questions
+      description: List all the unresolved questions so that they can be debated and considered. 
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Steps and milestones
+      
+  - type: textarea
+    attributes:
+      label: milestones
+      description: Provide key milestones so that the work steps for this Proposal are clear.
+
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Dependencies and problems
+
+  - type: markdown
+    attributes:
+      value: |
+        ### problems
+        
+  - type: textarea
+    attributes:
+      label: Potential Problems
+      description: Identify any potential issues or challenges that may arise during implementation.
+
+  - type: textarea
+    attributes:
+      label: Long-term Issues
+      description: Identify any long-term issues that might emerge after implementation.
+
+  - type: textarea
+    attributes:
+      label: Transient Problems
+      description: Identify any temporary issues arising from implementing the proposed solution.
+
+  - type: textarea
+    attributes:
+      label: impacting Branches
+      description: Detail any branches or repositories that the project will impact.
+
+  - type: markdown
+    attributes:
+      value: |
+        ### Dependencies
+
+  - type: textarea
+    attributes:
+      label: Dependencies
+      description: List all dependencies the project relies on, including software, hardware, and third-party services.
+
+  - type: textarea
+    attributes:
+      label: outgoing Dependencies
+      description: List all the projects, branches, and issues that rely on this implementation.
+
+  - type: textarea
+    attributes:
+      label: dependency Issues
+      description: Identify any existing issues that might impact the project.
```

Match capitalization to other labels
```suggestion
  - type: textarea
    attributes:
      label: Outgoing Dependencies
      description: List all the projects, branches, and issues that rely on this implementation.

  - type: textarea
    attributes:
      label: Dependency Issues
      description: Identify any existing issues that might impact the project.
```